Description: Fix build failure with GCC 4.6 (-Wunused-but-set-variable)
Author: Angel Abad <angelabad@ubuntu.com>
Bug-Ubuntu: https://launchpad.net/bugs/831182
Forwarded: no
Last-Update: 2011-08-23

--- a/gypsy/gypsy-time.c
+++ b/gypsy/gypsy-time.c
@@ -155,9 +155,9 @@
 	      GValue     *value,
 	      GParamSpec *pspec)
 {
-	GypsyTimePrivate *priv;
+	//GypsyTimePrivate *priv;
 
-	priv = GET_PRIVATE (object);
+	//priv = GET_PRIVATE (object);
 	switch (prop_id) {
 	case PROP_PATH:
 		break;
